examples/silentpayments.c

```diff
@@ -0,0 +1,432 @@
+/*************************************************************************
+ * Written in 2024 by josibake                                           *
```

Comment:
Our current recommendation is to skip all years and author info in new files (also in the C files), see the ellswift module for example. If there's no data, it cannot become wrong and outdated, and the git history anyway does a better job here. (And copyright notices are simply not required to obtain copyright.) But this is really just a recommendation, feel free to keep your name if you feel that attribution is a good idea.

Reply:
Fixed.